posthog/cdp/templates/_internal/template_blank.py

```diff
@@ -1,11 +1,11 @@
-from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+from posthog.cdp.templates.hog_function_template import HogFunctionMapping, HogFunctionTemplate
 
 blank_site_destination: HogFunctionTemplate = HogFunctionTemplate(
     status="client-side",
     type="site_destination",
     id="template-blank-site-destination",
     name="New client-side destination",
-    description="Run code on your website when an event is sent to PostHog. Works only with posthog-js when opt_in_site_apps is set to true.",
+    description="New destination with complex event mapping. Works only with posthog-js when opt_in_site_apps is set to true.",
```

Comment:
This is fine whilst we are testing internally but i think once we have a "real" destination using mappings we should revert this to be a simple single mapping function